tests/api/worlds/.robot_window_html.wbproj

```diff
@@ -0,0 +1,10 @@
+Webots Project File version R2020b
```

Comment:
Do we know why this particular one is un-ignored (https://github.com/cyberbotics/webots/blob/master/tests/api/worlds/.gitignore#L2) ?

Reply:
No idea. I believe we should remove it from the .gitignore.